corehq/apps/hqwebapp/utils/bootstrap/changes.py

```diff
@@ -267,3 +271,16 @@ def replace_path_references(filedata, old_reference, new_reference):
         repl=r"\1" + new_reference + r"\3",
         string=filedata
     )
+
+
+def update_gruntfile(filedata, mocha_paths):
+    mocha_apps = [path.replace('/spec', '').replace('/mocha.html', '')
```

Comment:
nit: could be a bit more defensive here to avoid erroneously replacing unintended matches like `/my/special/file` with `/myial/file`.
```suggestion
    mocha_apps = [path.replace('/spec/', '/').removesuffix('/mocha.html')
```